drivers/volume/portworx/portworx.go

```diff
@@ -1450,6 +1450,22 @@ func (d *portworx) UpdateIOPriority(volumeName string, priorityType string) erro
 	return nil
 }
 
+func (d *portworx) UpdateStickyFlag(volumeName, stickyOption string) error {
+	nodes := node.GetWorkerNodes()
```

Comment:
Please use getstoragedrivernodes, as in some cases like IKS there are worker nodes deployed for torpedo, with px disabled.